### DISCUSSION TITLE
Text input box is too short

### DISCUSSION THREAD

**Discussion by kfa99** [pnd280/complexity#48] [Q&A] [answered]
For some reason, the text input box has become too short. I haven't been able to fix this, even though I've asked AI many questions about it. Is there a way to fix this? I marked the problematic area with a red outline

![2024-11-24_14-25-34](https://github.com/user-attachments/assets/1b1d17e7-255d-446c-80b7-26b7197f81af)

**Comment by pnd280** [comment, marked as answer]
this issue is resolved, please follow the instruction here https://github.com/pnd280/complexity/issues/46#issuecomment-2495485590
